pkg/model/components/etcd.go

```diff
@@ -43,16 +45,35 @@ func (b *EtcdOptionsBuilder) BuildOptions(o interface{}) error {
 		}
 	}
 
-	// default to gcr.io
-	image := fmt.Sprintf("gcr.io/google_containers/etcd:%s", spec.EtcdClusters[0].Version)
+	// remap image
+	for _, c := range spec.EtcdClusters {
```

Comment:
Not sure why you are changing this in this PR.  Really is not dealing with the functionality of this PR, and seems to be fixing something else.  Thoughts?

Reply:
Fixes a bug - image remapping wasn't working previously.  And to have the fixed code right next to the not-fixed code looked weird.